recipes/astrometry.net/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "astrometry.net" %}
+{% set version = "0.78" %}
+
+package:
+  name: {{ name|lower}}
+  version: {{ version }}
+
+source:
+  fn: {{name}}-{{ version }}.zip
+  url: https://github.com/dstndstn/astrometry.net/archive/{{ version }}.zip
+  sha256: 85480b9636757c89ef5ccc7aac557409ac2ab377eb6c6eca862b16cdd4faefd9
+
+build:
+  number: 0
+  skip: True  # [osx or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - swig
+    - git
```

Comment:
Do you still need git?

Reply:
git not needed. Removed from the recipe.